src/solidation/__main__.py

```diff
@@ -250,7 +250,7 @@ def to_markdown(self) -> str:
                 if user is None:
                     user = i.user.login
                 s += (
-                    f"- [{i.title}]({i.html_url}) by {user}"
+                    f"- [{sanitize_md(i.title)}]({i.html_url}) by {user}"
```

Comment:
Presumably, you also want to do this to `i.title` on lines 274, 284, and 300 as well.

Reply:
@yarikoptic You missed `pr.title` on line 284 (now line 285).

Reply:
sharp eye!  I was searching for `i.title` and missed it. Thanks!